app/jobs/email.py

```diff
@@ -43,3 +43,41 @@ def send_confirm_email(email_address, token_url):
         server.close()
     except Exception as e:
         raise (e)
+
+
+@Q.job(timeout=60000)
+def send_password_change_confirm_email(email_address):
+    smtp_username = current_app.config["MAIL_USERNAME"]
+    smtp_password = current_app.config["MAIL_PASSWORD"]
+    host = current_app.config["MAIL_SERVER"]
+    port = current_app.config["MAIL_PORT"]
+    tls = current_app.config["MAIL_USE_TLS"]
+    msg = MIMEMultipart("alternative")
+    msg["Subject"] = "You've successfully changed your Holepunch password"
+    msg["From"] = email.utils.formataddr(("Holepunch NOREPLY", "noreply@holepunch.io"))
+    msg["To"] = email_address
+
+    body_text = "Your Holepunch account password was recently changed.  \n\nIf you didn't make this change, please let us know.  \n\nThanks! \n- The Holepunch Team"
+    body_html = f"""
+<html>
+<head></head>
+<body>
+ <h1>Holepunch Password Change Confirmation</h1>
+    </body>
+</html>
+"""
+    part1 = MIMEText(body_text, "plain")
+    part2 = MIMEText(body_html, "html")
+    msg.attach(part1)
+    msg.attach(part2)
+
+    try:
+        if tls:
+            server = smtplib.SMTP_SSL(host, port)
+        else:
+            server = smtplib.SMTP(host, port)
+        server.login(smtp_username, smtp_password)
+        server.sendmail("noreply@holepunch.io", email_address, msg.as_string())
+        server.close()
+    except Exception as e:
```

Comment:
We shouldn't catch `Exception` most of the time (especially if we're just going to re-raise it)  What's the actual error that might be thrown?